tests/test_wheel.py

```diff
@@ -31,9 +32,14 @@
 else:
     raise NotImplementedError(f'Unknown implementation: {platform.python_implementation()}')
 
+PLATFORM_TAG = sysconfig.get_platform().replace('-', '_').replace('.', '_')
```

Comment:
Actually, it might make sense to do  what [`cibuildwheel`](https://github.com/pypa/cibuildwheel/blob/bc3106a4d0cc1f1a604806f1e3795b0051125124/cibuildwheel/macos.py#L202) does for the target mac version for both cases.

Reply:
Perhaps, but I'd prefer to keep this simple until we need it. But that is good to keep in mind, thanks!